README.md

```diff
@@ -19,8 +19,10 @@ Yet it is very easy to use. You need learn only a few things.
 1. You have to always pass variables into the query via *placeholder*
 2. Each placeholder has to be marked with its data type. At the moment there are 6 types:
  * `?s` **("string")**              - strings (also `DATE`, `FLOAT` and `DECIMAL`)
+ * `?b` **("boolean")**             - booleans (can accept true, false, 1, 0, "1", or "0")
  * `?i` **("integer")**             - integers
- * `?n` **("name")**                - identifiers (table and field names) 
+ * `?n` **("name")**                - identifiers (table and field names)
+ * `?z` **("multi-field")**         - multiple field names (expects an array of values)
```

Comment:
- [ ] This isn't very clear. How about

  > Multi-identifier placeholder for listing column names (for example in a `SELECT` statement). Takes an array of identifiers; the placeholder will be subtituted for a string in ``` `field1`, `field2`, `field3` ``` format.